api/DOMImplementationList.json

```diff
@@ -1,148 +0,0 @@
-{
-  "api": {
-    "DOMImplementationList": {
```

Comment:
This went together with https://www.w3.org/TR/DOM-Level-3-Core/core.html#DOMImplementationSource, which isn't in BCD. This removal seems appropriate.